spring-pulsar-spring-boot-autoconfigure/src/main/java/org/springframework/pulsar/autoconfigure/PulsarProperties.java

```diff
@@ -1438,7 +1454,7 @@ public Map<String, String> getAuthParamMap() {
 			return this.authParamMap;
 		}
 
-		public void setAuthParamMap(Map<String, String> authParamMap) {
+		public void setAuthParamMap(SortedMap<String, String> authParamMap) {
```

Comment:
If we implement the change I suggested [above](https://github.com/spring-projects-experimental/spring-pulsar/pull/97/files#r967247500) where user can set the `authParams` directly, does SB respect the sorted contract? I would expect so, but just not 100% sure.